docs/getting-started/install-scylla/oss-install-aws.rst

```diff
@@ -0,0 +1,112 @@
+==========================
```

Comment:
Why id the file name oss-install-aws? in most cases there is nothing to install
a better name would be scylladb-on-aws or run-on-aws

example
https://www.mongodb.com/mongodb-on-aws